serializer.rst

```diff
@@ -1387,6 +1387,14 @@ normalizers (in order of priority):
     By default, an exception is thrown when data is not a valid backed enumeration. If you
     want ``null`` instead, you can set the ``BackedEnumNormalizer::ALLOW_INVALID_VALUES`` option.
 
+:class:`Symfony\\Component\\Serializer\\Normalizer\\NumberNormalizer`
+    This normalizer converts between :phpclass:`BcMath\Number` or :phpclass:`GMP` objects and
```

Comment:
Require https://github.com/symfony-tools/docs-builder/pull/190 in order to have correct generated link

Reply:
It was merged and Javier updated our dependencies 👐